kubectl-rook-ceph.sh

```diff
@@ -45,6 +45,9 @@ function print_usage() {
   echo "    status all                              : print the phase and conditions of all CRs"
   echo "    status <CR>                             : print the phase and conditions of CRs of a specific type, such as 'cephobjectstore', 'cephfilesystem', etc"
   echo "    purge-osd <osd-id> [--force]            : Permanently remove an OSD from the cluster. Multiple OSDs can be removed with a comma-separated list of IDs."
+  echo " start-debug <deployment-name> 
+                        [--image <alternate-image>] : Create the OSD or mon in debug deployment"
+  echo " stop-debug <deployment-name>               : Remove the OSD or mon out of debug deployment"
```

Comment:
What if we did `debug start` and `debug stop` instead of having two top-level commands? @travisn?

```suggestion
  echo " debug                                      : Debug a deployment by scaling it down and creating a debug copy. This is supported for mons and OSDs only
  echo "   start  <deployment-name>                 : Start debugging a deployment with an optional container image
  echo "     [--image <alternate-image>]"
  echo "   stop  <deployment-name>                  : Stop debugging a deployment
```

IMO, this allows us a little more space to document what debug mode is for a deployment. Otherwise, users may not know what to expect.